backend/azureblob/azureblob.go

```diff
@@ -233,6 +233,11 @@ This option controls how often unused buffers will be removed from the pool.`,
 				},
 			},
 			Advanced: true,
+		}, {
+			Name:     "no_head_object",
+			Help:     `If set, don't HEAD objects`,
```

Comment:
Can you expand the help here please

```go
`If set, don't HEAD objects

Consequences of not HEADing objects...
`,
```

I'm not sure of the consequences of not running HEAD and I'd like them spelled out in the docs please :-)

Reply:
Thank you for your review and comments.

I added help comments as follows.

`If set, don't HEAD objects. So, it will be faster when getting a lot of small files.`

Best regards,

Reply:
I think this means that if you use `no_head_object` then objects created directly with `NewObject` will not have

```
//  o.id
//  o.modTime
//  o.size
//  o.md5
```

What is your use case for this - how are you creating objects with `NewObject`?

Reply:
I changed help of this option.

`What is your use case for this - how are you creating objects with NewObject?`_

I use this option when the existence of many objects is certain. And I get the many objects with --no-head-objects --files-from-raw objects-list. Head also requires an API usage fee. Cost reduction is necessary.

Best regards,